pallets/stellar-relay/README.md

```diff
@@ -0,0 +1,38 @@
+# Stellar Relay Pallet
+
+## Testing
+
+To run the tests use:
+
+```bash
+cargo test --package pallet-stellar-relay --features runtime-benchmarks
```

Comment:
can we run the test without the feature `runtime-benchmarks`?

Reply:
Yes, you can run them without it (i.e. just `cargo test --package pallet-stellar-relay`). Adding the `runtime-benchmarks` feature will also make it run the test that is contained in 'benchmarks.rs' (which doesn't do much).